opm/autodiff/NonlinearSolver_impl.hpp

```diff
@@ -137,7 +137,11 @@ namespace Opm
             }
             converged = report.converged;
             linIters += report.linear_iterations;
-            wellIters += report.well_iterations;
+            if (report.well_iterations != std::numeric_limits<int>::min()) {
+                wellIters += report.well_iterations;
+            } else {
+                wellIters = report.well_iterations;
```

Comment:
``report.well_iterations`` is initialized with ``std::numeric_limits<int>`` in all the place, if ``solveWellEq()`` is not called, it will stay the same, I think.